libraries/host_vswitch.rb

```diff
@@ -16,68 +16,58 @@ class VmWareHostVswitch < Inspec.resource(1)
 
   # Load the configuration file on initialization
   def initialize(opts)
-    @opts = opts;
+    @opts = opts
   end
 
   # Expose all parameters
-  def method_missing(name)
-    return forgedTransmits[name.to_s]
+  def method_missing(name) # rubocop:disable Style/MethodMissing
+    forgedTransmits[name.to_s]
   end
 
-  def forgedTransmits
```

Comment:
As this is used to call the method I thought that the name should be like the method.